docs/source/api/mobile-config.md

```diff
@@ -31,8 +31,8 @@ App.icons({
 });
 
 App.launchScreens({
-  'iphone_2x': 'splash/Default@2x~iphone.png',
-  'iphone5': 'splash/Default~iphone5.png',
+  'ios_universal': { src: 'splash/Default@2x.png', srcDarkMode: 'splash/Default@2x~dark.png' },
```

Comment:
I believe this doc could show the whole list of options and then we could link it in the breaking change as well. So people could see all the options easily.

I would also add a section below saying
> This changed in Meteor version 2.6.0, previous we had etc, etc, etc

What do you think?

It's a nice opportunity to improve this document.

Could you also send a PR updating this [README](https://github.com/meteor/examples/tree/main/cordova#readme)? I know many people use it to configure their apps.

Reply:
@filipenevola Yeah,that makes sense. Will update this README you linked too.

Just one thing, about this part:

> I would also add a section below saying

>>This changed in Meteor version 2.6.0, previous we had etc, etc, etc

The only place that I think that we could add something similar to this would be as part of the `Migration Steps` on the changelog, would that be OK? But still, I don't really think it's necessary, as it is something that broke on 2.2 (with the cordova 10 update).

Reply:
@filipenevola Applied the changes you asked here, please, review again! Also, will create a PR on the cordova example repository.